docs/reference/query-dsl/geo-polygon-query.asciidoc

```diff
@@ -80,6 +84,7 @@ GET /_search
   }
 }
 --------------------------------------------------
+// TEST[warning:Deprecated field [geo_polygon] used, replaced by [[geo_shape] query where polygons are defined in geojson or wkt]]
```

Comment:
Is the `geo_polygon` is a different query from `geo_shape`? 
Or is it the same query and it was just renamed from `geo_polygon` to `geo_shape`?

This warning looks like we just renamed the query.  I thought we use this kind of warning when we rename the fields.

Reply:
It is a different query that has the same functionality (and much more) that the geo_polygon query.  I try to follow the same strategy as here: https://github.com/elastic/elasticsearch/pull/42619.

Reply:
@iverase Thanks for the clarification, I was not aware that we use `withAllDeprecated` also with queries!